### ISSUE TITLE
Default Status when creating task from api

### ISSUE THREAD

**Issue by rlecaro2** [taigaio/taiga-back#309]
When creating a User Story the default status is 'New' if not provided. This isn't the case when doing the same with tasks and, in my opinion, should behave the same way.

**Comment by bameda** [comment]
No because task statuses are different from stories statuses.

In the API you have resources to manage all of then:

 - US Statuses resource: http://taigaio.github.io/taiga-doc/dist/api.html#user-story-statuses
 - Task Statuses resource: http://taigaio.github.io/taiga-doc/dist/api.html#task-statuses
 - Issue Statuses resource: http://taigaio.github.io/taiga-doc/dist/api.html#issue-statuses

You can set defaults statuses for new user stories (default_us_status attribute of a project object), task (default_task_status attribute of a project object) or Issues (default_issue_status attribute of a project object) editing the project. An example:

```
curl -X PUT \
  -H "Content-Type: application/json" \
  -H "Authorization: Bearer ${AUTH_TOKEN}" \
  -d '{
            "default_us_status": 1,
            "default_task_status": 1,
            "default_issue_status": 1,
      }' \
  https://api.taiga.io/api/v1/projects/1
```
> in the example `1` is an object id

**Comment by rlecaro2** [comment]
I understand. Nevertheless the status field is required when creating tasks instead of using its default when not provided. Is it intended?

**Comment by narisso** [comment]
+ 1

**Comment by superalex** [comment]
Hello @rlecaro2, it's intented to use the default specified status tasks if your are not specified any one in the API creation call. You can also customize these values in the web app selecting your project and going to admin -> project -> default values.

Regards,

**Comment by rlecaro2** [comment]
Hello @superalex, that's my problem, I'm getting {"status": ["This field is required."]} as a response from the API when not providing it. This happens only creating tasks, not user stories.

**Comment by superalex** [comment]
Thanks @rlecaro2, I think it's a bug, it should be working in the same way in both cases, we'll check try to solve it as soon as possible.

**Comment by jespino** [comment]
Fixed in the PR #368 this will be included in the next release :smile:
